hr_family/models/hr_employee.py

```diff
@@ -19,20 +19,27 @@
 #
 #
 
-from openerp import models, fields
+from odoo import models, fields
 
 
 class HrEmployee(models.Model):
     _inherit = 'hr.employee'
 
-    fam_spouse = fields.Char("Name")
-    fam_spouse_employer = fields.Char("Employer")
-    fam_spouse_tel = fields.Char("Telephone.")
+    fam_spouse = fields.Char(string="Spouse's Name")
```

Comment:
@GillesTephaneMeyomesse As these fields is in a group with a string which is spouse, I think that is redundant to add spouse in the label. Any other opinion ?